backend/schema.sql

```diff
@@ -191,3 +191,19 @@ CREATE TABLE redeployment_events (
 	message TEXT,
 	created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
+
+CREATE TABLE notice (
+	id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+	path VARCHAR(255) NOT NULL,
```

Comment:
`path`はお知らせの外部 ID と見ることができて、一意制約を満たすのではないか
```suggestion
	path VARCHAR(255) NOT NULL UNIQUE,
```